systemd/x0vncserver.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=Remote desktop service (VNC) for :0 display
 Requires=display-manager.service
+After=network-online.target
```

Comment:
this has never made a n appeareance here as it was causing very long start times when not connected to the net

Reply:
Indeed, this causes a longer startup, but this is necessary for the vnc dummy server configuration to be taken into account. Without this, the geometry is not carried out by x0vncserver and the display becomes a square geometry screen
The additional time taken at the start remains acceptable compared to the gain obtained by activating all the graphics options of the driver

Reply:
I am not sold. How the network makes this working? Network should have nothing to do with video, maybe we should find the right dependencies here?

Reply:
I actually think the problem is a tigervnc related issue directly, not the video server. In the wiki, they build a systemd service with After=network-online.target.
When I didn't use this line, x0vncserver displayed a square virtual screen. After adding, the configuration is taken into account by x0vncserver. I imagine that there is in tigervnc that if it does not obtain a fast enough response from the network, it launches a server without configuration with square display. As the pi becomes a little slower now with the new video driver, we encounter this problem

Reply:
I did not test by removing this line to see if the pi 5 which is faster encountered the same issue